casper/src/main/scala/coop/rchain/casper/merging/DagMerger.scala

```diff
@@ -15,12 +15,16 @@ import coop.rchain.shared.Log
 
 object DagMerger {
 
+  def costOptimalRejectionAlg: DeployChainIndex => Long =
+    (r: DeployChainIndex) => r.deploysWithCost.map(_.cost).sum
+
   def merge[F[_]: Concurrent: Log](
       dag: BlockDagRepresentation[F],
       lfb: BlockHash,
       lfbPostState: Blake2b256Hash,
       index: BlockHash => F[Vector[DeployChainIndex]],
-      historyRepository: RhoHistoryRepository[F]
+      historyRepository: RhoHistoryRepository[F],
+      optimalRejectionAlg: DeployChainIndex => Long
```

Comment:
@nzpr @tgrospic Sorry. I still expose this method out. 
It would be a problem for some random generated test cases that I don't have to way to decide which part I want to reject.

Reply:
I'm ok with that, just imo its better to call it `rejectionCost`